src/compas_timber/ghpython/components/CT_Beams_From_Mesh/code.py

```diff
@@ -0,0 +1,75 @@
+"""Creates a Beam from a LineCurve."""
+
+import rhinoscriptsyntax as rs
+from compas.geometry import Line
+from compas.geometry import Vector
+from compas.scene import Scene
+from ghpythonlib.componentbase import executingcomponent as component
+from Grasshopper import DataTree
+from Grasshopper.Kernel.Data import GH_Path
+from Grasshopper.Kernel.GH_RuntimeMessageLevel import Warning
+
+from compas_timber.elements import Beam as CTBeam
```

Comment:
why the renaming?

Reply:
it is like this in `Beam_from_curve` component...